jolt-core/src/main/java/com/bazaarvoice/jolt/templatr/spec/TemplatrFunctionLeafSpec.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2013 Bazaarvoice, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bazaarvoice.jolt.templatr.spec;
+
+import com.bazaarvoice.jolt.common.Optional;
+import com.bazaarvoice.jolt.common.SpecStringParser;
+import com.bazaarvoice.jolt.common.tree.MatchedElement;
+import com.bazaarvoice.jolt.common.tree.WalkedPath;
+import com.bazaarvoice.jolt.exception.SpecException;
+import com.bazaarvoice.jolt.templatr.OpMode;
+import com.bazaarvoice.jolt.templatr.TemplatrSpecBuilder;
+import com.bazaarvoice.jolt.templatr.function.Function;
+import com.bazaarvoice.jolt.templatr.function.FunctionArg;
+import com.bazaarvoice.jolt.templatr.function.Strings;
+import com.bazaarvoice.jolt.templatr.function.Math;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TemplatrFunctionLeafSpec extends TemplatrSpec {
+
+    private static final Map<String, Function> BUILT_INS = new HashMap<>(  );
+
+    static {
+        BUILT_INS.put( "toLower", new Strings.toLowerCase() );
+        BUILT_INS.put( "toUpper", new Strings.toUpperCase() );
+        BUILT_INS.put( "toProper", new Strings.toProperCase() );
+        BUILT_INS.put( "strConcat", new Strings.strConcat() );
```

Comment:
strConcat -> concat   
Does not make sense to concat anything other than Strings.